config/webpack.config.dev.js

```diff
@@ -65,6 +65,9 @@ module.exports = {
     publicPath: '/'
   },
   resolve: {
+    // This allows you to set a root for where webpack should look for modules.
+    // This enables you to use absolute imports from the root.
+    root: path.resolve(env['process.env.NODE_PATH']),
```

Comment:
We can probably use just `process.env.NODE_PATH` itself here without `env`, what do you think?